asyncapi-cli/src/main/java/io/ballerina/asyncapi/cmd/websockets/CmdConstants.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.asyncapi.cmd.websockets;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Constants for asyncapi commands.
+ *
+ * @since 1.2.0
```

Comment:
why do we have different `since` value ?

Reply:
Fixed this in https://github.com/ballerina-platform/asyncapi-tools/pull/119/commits/72dabb8aa34cc83745f989834bc2c8a8b555a5a7